src/Video.js

```diff
@@ -0,0 +1,15 @@
+import React from "react";
+import './Video.css'
```

Comment:
Decide between `' '` or `" "` and whether to end statements with a semi-colon or not. :)